.env.example

```
@@ -0,0 +1 @@
+VITE_API_BEARER_TOKEN=abc
```

No need to add a random value. The `.env.example` should have valid values, for example, in case you want to add development values for things that are not secret like the port used by your app, or no value at all, just the variable name.
```suggestion
VITE_API_BEARER_TOKEN=
```

---

Just committed an updated version with no value